samples/nrf9160/asset_tracker/src/main.c

```diff
@@ -635,6 +691,15 @@ static void button_handler(u32_t buttons, u32_t has_changed)
 		flip_send(NULL);
 	}
 
+	if (IS_ENABLED(CONFIG_CLOUD_BUTTON) &&
+	   (has_changed & CONFIG_CLOUD_BUTTON_INPUT)) {
+		if (buttons & CONFIG_CLOUD_BUTTON_INPUT) {
```

Comment:
Can be simplified to `button_send(buttons & CONFIG_CLOUD_BUTTON_INPUT)` to get rid of the `if` / `else`